test/contract.test.js

```diff
@@ -56,6 +56,32 @@ const contract = new Contract(account, null, {
     });
 });
 
+describe('viewMethod', () => {
+    test('passes options through to account viewFunction', async () => {
+        function customParser () {}
+        const stubbedReturnValue = await contract.viewMethod({}, { parse: customParser });
+        expect(stubbedReturnValue.options.parse).toBe(customParser);
+    });
+
+    test('throws PositionalArgsError if second argument is not an object', () => {
+        return Promise.all([
```

Comment:
nit: seems like a use case for https://jestjs.io/docs/en/api#describeeachtablename-fn-timeout